README.md

```diff
@@ -75,7 +75,7 @@ The components within this repository are delivered as multi-targeted Nuget pack
 
 Equinox does not focus on projection logic or wrapping thereof - each store brings its own strengths, needs, opportunities and idiosyncrasies. Here's a list of some relevant libraries from sibling projects that get used with regard to this:
 
-- `Jet.ConfluentKafka.FSharp` [![Jet.ConfluentKafka.FSharp NuGet](https://img.shields.io/nuget/vpre/Jet.ConfluentKafka.FSharp.svg)](https://www.nuget.org/packages/Jet.ConfluentKafka.FSharp/): Wraps `Confluent.Kafka` to provide efficient batched Kafka Producer and Consumer configurations, with basic logging instrumentation. Used in the [`propulsion project kafka`](dotnet-tool-provisioning--benchmarking-tool) tool command; see [`dotnet new proProjector -k; dotnet new proConsumer` to generate a sample app](quickstart) using it (see the `BatchedAsync` and `BatchedSync` modules in `Examples.fs`).
+- `Jet.ConfluentKafka.FSharp` [![Jet.ConfluentKafka.FSharp NuGet](https://img.shields.io/nuget/vpre/Jet.ConfluentKafka.FSharp.svg)](https://www.nuget.org/packages/Jet.ConfluentKafka.FSharp/): Wraps `Confluent.Kafka` to provide efficient batched Kafka Producer and Consumer configurations, with basic logging instrumentation. Used in the [`propulsion project kafka`](dotnet-tool-provisioning--benchmarking-tool) tool command; see [`dotnet new proProjector -k; dotnet new proConsumer` to generate a sample app](https://github.com/jet/dotnet-templates#propulsion-related) using it (see the `BatchedAsync` and `BatchedSync` modules in `Examples.fs`).
```

Comment:
> [`propulsion project kafka`](dotnet-tool-provisioning--benchmarking-tool)

should probably link to https://github.com/jet/propulsion#dotnet-tool-provisioning--projections-test-tool (there is no doc in there that describes the tool's sub-commands as such)

Reply:
i am reading documentation, will update what will found.

Reply:
Thanks - I for one really appreciate those sorts of things being picked up
And a heads up, I've just released nugets for all and sundry in the last few hours if you're playing about